README.md

```diff
@@ -19,10 +19,18 @@ PowerShell version of [asciinema](https://github.com/asciinema/asciinema) based
 
 ## Installation
 
+### Cargo
+
 ```console
 cargo install PowerSession
 ```
 
+### Scoop
+
+```console
+scoop install powersession-rs
```

Comment:
Is it a convention that the package name all be lowercases ?

Reply:
That seems to be the case for official buckets.